jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/SeleniumTestHelper.java

```diff
@@ -28,36 +41,45 @@
 
     /**
      * Create an instance of a {@link WebDriver} to facilitate browser based testing
+     *
      * @param webDriverClass The name of the web driver class to use
      * @return {@link WebDriver}
      */
-    public static WebDriver createWebDriver(final String webDriverClass) {
+    public static WebDriver createWebDriver(final String webDriverClass) throws IOException {
         final WebDriver driver;
-        try {
-            driver = (WebDriver) Class.forName(webDriverClass).getConstructor().newInstance();
-        } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException | ClassNotFoundException e) {
-            throw new SeleniumTestCaseException(e);
-        }
+
+        final DesiredCapabilities dc = DesiredCapabilities.internetExplorer();
+        dc.setCapability(CapabilityType.UNEXPECTED_ALERT_BEHAVIOUR, UnexpectedAlertBehaviour.DISMISS);
+        dc.setCapability(CapabilityType.TAKES_SCREENSHOT, true);
+        dc.setCapability(CapabilityType.SUPPORTS_JAVASCRIPT, true);
+        dc.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
+        String hubUrl = (String) getProperties().getProperty("hub.url");
+        hubUrl = hubUrl.replaceAll("\"", "");
```

Comment:
Missed reviewing this one my bad!

Reply:
Looks like Selenium Grid is only used in IE...if I were to code it it would be for both my bad again I missed it